src/sentry/static/sentry/app/components/events/contexts/getUnknownData.tsx

```diff
@@ -0,0 +1,19 @@
+import {KeyValueListData} from 'app/components/events/interfaces/keyValueList/types';
+import {getMeta} from 'app/components/events/meta/metaProxy';
+
+function getUnknownData(
```

Comment:
Might be a good idea to add a unit test for this.

Reply:
True true, added.